src/main/java/net/pms/PMS.java

```diff
@@ -1368,18 +1375,46 @@ public static int getOSBitness() {
 		return bitness;
 	}
 
+	private int cpuScore;
+
+	/**
+	 * Returns the CPU benchmark score which can be used to determine the transcoding setting
+	 * in accordance with the CPU power. If null than the CPU is unknown or not benchmarked.
+	 */
+	public int getCpuScore() {
+		return cpuScore;
+	}
+
 	/**
 	 * Log system properties identifying Java, the OS and encoding and log
 	 * warnings where appropriate.
 	 */
 	private void logSystemInfo() {
 		long memoryInMB = Runtime.getRuntime().maxMemory() / 1048576;
+		HardwareAbstractionLayer hal = new SystemInfo().getHardware();
```

Comment:
We should probably wrap this in getExternalNetwork(), and feel free to rename that to isExternalNetwork too, or I can